EpikCord/__main__.py

```diff
@@ -1,5 +1,39 @@
-from EpikCord import __version__
 
+import argparse
+import colorama
+
+
+__all__ = ("__version__", "info")
+
+__version__ = "0.5.2"
+
+
+_bot_tmplate = """#!/usr/bin/env python
+from EpikCord import Client, Intents
+
+intents = Intents.all()
+
+client: Client = Client([token], intents)
+
+
+@client.event()
+async def ready():
+    print("Ready!")
+# Write commands here
+
+client.login()
+"""
+
+colorama.init(autoreset=True)
```

Comment:
If people wanted to see the version, they could do `pip freeze` or use interactive mode to import `__version__`

Reply:
why not just display the version?